src/main/java/randoop/main/GenTests.java

```diff
@@ -159,6 +165,32 @@
   /** The count of sequences that failed to compile. */
   private int sequenceCompileFailureCount = 0;
 
+  /** TypedOperationHeuristic is a wrapper around a TypedOperation and a heuristic. */
+  private static class TypedOperationHeuristic {
```

Comment:
This seems like the stylistically correct Java way to create a generic type for a PriorityQueue, but it seems a bit heavy-handed.